src/ts/components/subtitleoverlay.ts

```diff
@@ -457,22 +483,27 @@ export class SubtitleRegionContainerManager {
     this.subtitleOverlay = subtitleOverlay;
   }
 
+  private getRegion(label: SubtitleLabel): { regionContainerId: string, regionName: string } {
+    if (label.vtt) {
+      return {
+        regionContainerId: label.vtt.region && label.vtt.region.id ? label.vtt.region.id : 'vtt',
+        regionName: 'vtt',
+      };
+    } else {
+      return {
+        regionContainerId: label.region || 'default',
+        regionName: label.region || 'default',
+      };
+    }
```

Comment:
No need for ELSE as we have a return in the previous statement
```suggestion
    }
      return {
        regionContainerId: label.region || 'default',
        regionName: label.region || 'default',
      };
   
```

Reply:
I agree, fixed in https://github.com/bitmovin/bitmovin-player-ui/pull/378/commits/6b9d35304a117fb79d93fe0fad303c7e013b989d